ElectronClient/app.js

```diff
@@ -406,11 +407,13 @@ class Application extends BaseApplication {
 		const sortNoteItems = sortNoteFolderItems('notes');
 		const sortFolderItems = sortNoteFolderItems('folders');
 
+		const keymapService = KeymapService.instance();
+
 		const focusItems = [
-			cmdService.commandToMenuItem('focusElementSideBar', 'CommandOrControl+Shift+S'),
-			cmdService.commandToMenuItem('focusElementNoteList', 'CommandOrControl+Shift+L'),
-			cmdService.commandToMenuItem('focusElementNoteTitle', 'CommandOrControl+Shift+N'),
-			cmdService.commandToMenuItem('focusElementNoteBody', 'CommandOrControl+Shift+B'),
+			cmdService.commandToMenuItem('focusElementSideBar', keymapService.getAccelerator('focusElementSideBar')),
+			cmdService.commandToMenuItem('focusElementNoteList', keymapService.getAccelerator('focusElementNoteList')),
+			cmdService.commandToMenuItem('focusElementNoteTitle', keymapService.getAccelerator('focusElementNoteTitle')),
+			cmdService.commandToMenuItem('focusElementNoteBody', keymapService.getAccelerator('focusElementNoteBody')),
```

Comment:
That's a lot of duplicated code. I think the KeymapService should be passed as a dependency to the CommandService, then in `CommandService::commandToMenuItem` you can set the accelerator there directly. That way instead of `cmdService.commandToMenuItem('focusElementSideBar', keymapService.getAccelerator('focusElementSideBar')),` you'd have just `cmdService.commandToMenuItem('focusElementSideBar)`.

It's the purpose of this commandToMenuItem function - to reduce boiler plate code when creating a menu item, and that should include setting up the accelerator.

Reply:
I've added it to commandToMenuItem. Please have a look.

Since not all menu items have keyboard shortcuts, I've added an extra check to verify that the `commandName` is in the current keymap so that it'll not call `getAccelerator()` if that command has no shortcut, preventing an unnecessary try catch block. There's no "error" to catch in such scenario because it's not really an error.  But I'll remove that if it's unnecessary.